docs/en/stack/ml/df-analytics/ml-dfa-feature-processors.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[ml-feature-processors]]
+= {dfanalytics-cap} feature processors
+++++
+<titleabbrev>Feature processors</titleabbrev>
+++++
+
+beta::[]
+
+Feature processors enable you to extract process features from raw document 
+fields of your index. These features can be used in model training and 
+inference. The resulting features are ephemeral; they are not stored in the 
+index. This provides a mechanism to create features that can be used at search 
+and ingest time and don't take up space in the index. Additionally, it 
+simplifies model management as both the features and the model can follow the 
+same life cycle. Automatic <<ml-feature-encoding, categorical encoding>> still 
+occurs during the analytics process even if custom feature processors are used. 
+Refer to the `feature_processors` property of the 
+{ref}/put-dfanalytics.html[Create {dfanalytics-job} API] to learn more.
+
+Available feature processors:
+
+* <<frequency-encoding>>
+* <<multi-encoding>>
+* <<ngram-encoding>>
+* <<one-hot-encoding>>
+* <<target-mean-encoding>>
+
+
+[[frequency-encoding]]
+== Frequency encoding
+
+Frequency encoding takes into account how many times a given categorical feature 
+is present in relation to a particular value in the data set. The more 
```

Comment:
This has to do with the value of the field that is being encoded. Not just any particular value in the data set. We should make the relationship clearer.